packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -329,6 +329,23 @@ export class StripeHelper {
     );
   }
 
+  async invoicePayableWithPaypal(invoice: Stripe.Invoice): Promise<boolean> {
+    if (invoice.billing_reason === 'subscription_create') {
+      // We only work with non-creation invoices, initial invoices are resolved by
+      // checkout code.
+      return false;
+    }
+    const subscription = await this.expandResource(
+      invoice.subscription,
+      'subscriptions'
+    );
+    if (subscription?.collection_method !== 'send_invoice') {
+      // Not a PayPal funded subscription.
+      return false;
+    }
+    return true;
+  }
```

Comment:
This looks fine, and I don't know what the future plans are for subplat... but there's nothing PayPal specific here.  I guess I just wish 'send_invoice' and payable with PayPal aren't effectively equivalent. :see_no_evil: